<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.0.1 (Python 3.9.6 on darwin)" generated="2024-08-17T03:57:23.440629" rpa="false" schemaversion="5">
<suite id="s1" name="TestSuite" source="/Users/andri.gunawan/PycharmProjects/Lionnn/TestCase/TestSuite.robot">
<test id="s1-t1" name="Start" line="19">
<kw name="Open Application" owner="AppiumLibrary">
<kw name="Capture Page Screenshot" owner="AppiumLibrary">
<msg time="2024-08-17T03:57:23.564295" level="FAIL">No application is open</msg>
<doc>Takes a screenshot of the current page and embeds it into the log.</doc>
<status status="FAIL" start="2024-08-17T03:57:23.564082" elapsed="0.007454">No application is open</status>
</kw>
<msg time="2024-08-17T03:57:23.572231" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
<msg time="2024-08-17T03:57:23.572329" level="FAIL">WebDriverException: Message: The requested resource could not be found, or a request was received using an HTTP method that is not supported by the mapped resource
</msg>
<arg>${url}</arg>
<arg>platformName=${platformName}</arg>
<arg>platformVersion=${platformVersion}</arg>
<arg>deviceName=${deviceName}</arg>
<arg>appPackage=${appPackage}</arg>
<arg>automationName=${automationName}</arg>
<arg>noReset=true</arg>
<arg>doesNoReset=true</arg>
<arg>appActivity=${appActivity}</arg>
<doc>Opens a new application to given Appium server.
Capabilities of appium server, Android and iOS,
Please check https://appium.io/docs/en/2.1/cli/args/
| *Option*            | *Man.* | *Description*     |
| remote_url          | Yes    | Appium server url |
| alias               | no     | alias             |
| strict_ssl          | No     | allows you to send commands to an invalid certificate host like a self-signed one. |</doc>
<status status="FAIL" start="2024-08-17T03:57:23.550379" elapsed="0.022036">WebDriverException: Message: The requested resource could not be found, or a request was received using an HTTP method that is not supported by the mapped resource
</status>
</kw>
<kw name="Sleep" owner="BuiltIn">
<arg>3</arg>
<doc>Pauses the test executed for the given time.</doc>
<status status="NOT RUN" start="2024-08-17T03:57:23.572523" elapsed="0.000015"/>
</kw>
<status status="FAIL" start="2024-08-17T03:57:23.549808" elapsed="0.022813">WebDriverException: Message: The requested resource could not be found, or a request was received using an HTTP method that is not supported by the mapped resource
</status>
</test>
<status status="FAIL" start="2024-08-17T03:57:23.441797" elapsed="0.131049"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" skip="0" id="s1" name="TestSuite">TestSuite</stat>
</suite>
</statistics>
<errors>
<msg time="2024-08-17T03:57:23.458799" level="ERROR">Error in file '/Users/andri.gunawan/PycharmProjects/Lionnn/TestCase/TestSuite.robot' on line 3: Importing library 'AppiumLibrar' failed: ModuleNotFoundError: No module named 'AppiumLibrar'
Traceback (most recent call last):
  None
PYTHONPATH:
  /Users/andri.gunawan/PycharmProjects/Lionnn
  /Users/andri.gunawan/PycharmProjects/Lionnn
  /Applications/PyCharm.app/Contents/plugins/python-ce/helpers/pycharm_display
  /Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python39.zip
  /Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9
  /Applications/Xcode.app/Contents/Developer/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/lib-dynload
  /Users/andri.gunawan/PycharmProjects/Lionnn/.venv/lib/python3.9/site-packages
  /Applications/PyCharm.app/Contents/plugins/python-ce/helpers/pycharm_matplotlib_backend
  /Applications/PyCharm.app/Contents/plugins/python-ce/helpers/pycharm_plotly_backend</msg>
<msg time="2024-08-17T03:57:23.572231" level="WARN">Keyword 'Capture Page Screenshot' could not be run on failure: No application is open</msg>
</errors>
</robot>
